internal/specs/compliance/nsa-1.0.yaml

```diff
@@ -0,0 +1,175 @@
+      severity: 'MEDIUM'
+    - name: Namespace kube-system should not be used by users
+      description: 'Control check whether Namespace kube-system is not be used by users'
+      id: '1.12'
+      defaultStatus: 'FAIL'
+      checks:
+        - id: AVD-KSV-0037
+      severity: 'MEDIUM'
+    - name: Pod and/or namespace Selectors usage
+      description: 'Control check validate the pod and/or namespace Selectors usage'
+      id: '2.0'
+      defaultStatus: 'FAIL'
+      checks:
+        - id: AVD-KSV-0038
+      severity: 'MEDIUM'
+    - name: Use CNI plugin that supports NetworkPolicy API
+      description: 'Control check whether check cni plugin installed'
+      id: '3.0'
+      defaultStatus: 'FAIL'
+      checks:
+      severity: 'CRITICAL'
+    - name: Use ResourceQuota policies to limit resources
+      description: 'Control check the use of ResourceQuota policy to limit aggregate resource usage within namespace'
+      id: '4.0'
+      defaultStatus: 'FAIL'
+      checks:
+        - id: AVD-KSV-0040
+      severity: 'MEDIUM'
+    - name: Use LimitRange policies to limit resources
+      description: 'Control check the use of LimitRange policy limit resource usage for namespaces or nodes'
+      id: '4.1'
+      defaultStatus: 'FAIL'
+      checks:
+        - id: AVD-KSV-0039
+      severity: 'MEDIUM'
+    - name: Control plan disable insecure port
+      description: 'Control check whether control plan disable insecure port'
+      id: '5.0'
+      defaultStatus: 'FAIL'
+      checks:
+      severity: 'CRITICAL'
+    - name: Encrypt etcd communication
+      description: 'Control check whether etcd communication is encrypted'
+      id: '5.1'
+      checks:
+        - id: AVD-KCV-0030
+      severity: 'CRITICAL'
+    - name: Ensure kube config file permission
+      description: 'Control check whether kube config file permissions'
+      id: '6.0'
+      defaultStatus: 'FAIL'
+      checks:
+      severity: 'CRITICAL'
+    - name: Check that encryption resource has been set
+      description: 'Control checks whether encryption resource has been set'
+      id: '6.1'
+      checks:
+        - id: AVD-KCV-0029
+      severity: 'CRITICAL'
+    - name: Check encryption provider
+      description: 'Control checks whether encryption provider has been set'
+      id: '6.2'
+      checks:
+        - id: AVD-KCV-0004
+      severity: 'CRITICAL'
+    - name: Make sure anonymous-auth is unset
+      description: 'Control checks whether anonymous-auth is unset'
+      id: '7.0'
+      checks:
+        - id: AVD-KCV-0001
+      severity: 'CRITICAL'
+    - name: Make sure -authorization-mode=RBAC
+      description: 'Control check whether RBAC permission is in use'
+      id: '7.1'
+      checks:
+        - id: AVD-KCV-0008
+      severity: 'CRITICAL'
+    - name: Audit policy is configure
+      description: 'Control check whether audit policy is configure'
+      id: '8.0'
+      defaultStatus: 'FAIL'
+      checks:
```

Comment:
Another quick q: why some of these checks don't have an associated rule (check)? Is it because we don't support it ?

Reply:
Yes , see #1042 ,we now tag it as Manual checks. Trivy can't run these checks in a non intrusive mode (outside of the cluster) .but it still part of Nsa so we added it.